src/database.ts

```diff
@@ -1991,7 +1993,7 @@ class Database extends GrpcServiceObject {
 
     const getWriteSession = this.pool_.getWriteSession.bind(this.pool_);
     // Loop to retry 'Session not found' errors.
-    while (true) {
+    for (;;) {
```

Comment:
That is far less obvious.  Can we put it back?

Reply:
Nope - it's an `eslint` rule :) Do you want to make a `// eslint-ignore` exception here?

Reply:
I think that would be better?  But I'm not blocking on it or anything :)

Reply:
Reverted!